GDJS/Runtime/howler-sound-manager/howler-sound-manager.ts

```diff
@@ -932,6 +986,93 @@ namespace gdjs {
       }
     }
 
+    getNetworkSyncData(): SoundManagerSyncData {
+      const freeMusicsNetworkSyncData: SoundSyncData[] = this._freeMusics
+        .map((freeMusic) => freeMusic.getNetworkSyncData())
+        .filter((musicSyncData) => !!musicSyncData);
```

Comment:
Ubernit (slightly more concise and readable when you scan the code):
```suggestion
        .filter(Boolean);
```

Reply:
Yeah for some reason, TS isn't happy with that. I changed the logic altogether so it's happy now